app/Repos/UserRepo.php

```diff
@@ -145,6 +148,12 @@ public function destroy(User $user)
     {
         $user->socialAccounts()->delete();
         $user->delete();
+        
+        // Deleting User profile pics
+        $profilePic = $user->image_id ? $user->avatar->findOrFail($user->image_id) : FALSE;
+        if ($profilePic) {
+            $this->imageService->destroyImage($profilePic);
```

Comment:
With the services, Instead of injecting it in via the constructor I've currently been using them via Facades when only sparingly used. Would be good to keep it this way to align with rest of image use in this Repo:
```
\Images::destroyImage($profilePic);
```